bot/commands/misc/avatar.js

```diff
@@ -18,6 +18,6 @@ module.exports = new Command('avatar', async (msg, args) => {
 
 // TODO: add args
 module.exports.help = {
-	args: '',
+	args: '<Discord user name>',
```

Comment:
We should probably try to keep terminology consistent for where Discord users are passed. Let's just go with `<user>` to denote a Discord user, since any of the existing commands will parse a user from a mention, username#discrim tag, or plain ID. Doesn't need repeating in every command's help page.

```suggestion
	args: '<user>',
```